commons/connector/health/health_activity.avsc

```diff
@@ -0,0 +1,24 @@
+{
+  "namespace": "org.radarcns.connector.health",
+  "name": "HKWorkoutTypeIdentifier_HKCategoryTypeIdentifierSleepAnalysis",
```

Comment:
The package identifies that this is about HealthKit, the fact that this is a schema gives it a type identifier. So adding `TypeIdentifier` to the name is redundant. Alternative proposal: `HealthKitActivity`